Configuration_adv.hpp

```diff
@@ -474,25 +480,40 @@
 
 // RA EndSwitch support
 //////////////////////////////////////////
+// End switches allow you to define how many percent of the distance needed to stop
```

Comment:
I'd also add something about what "percent" means. Like how does it correspond to physical movement? When first reading I expected it to be either steps or hours etc. Some sort of comment explanation would help a lot.

Reply:
Like how is the percentage calculated? Percent is a relative measure between two values - what values are those in this case? It's not clear to me

Reply:
Hmmm, ok let me see if I can explain it and then you can maybe word it better. 
When the switch triggers, it takes a certain distance until we stop (deceleration ramp). The number of steps needed to stop is the 100% value. We then slew in the opposite direction by this number of steps times the given percentage.

Reply:
We could also specify this in degrees from the trigger point. And that might actually make more sense, since then we are not dependent on the speed and thus the length of the ramp. Since the hysteresis of a switch is constant, degrees are probably better anyway. I'll change that.